docker/Dockerfile

```diff
@@ -1,20 +1,56 @@
-FROM golang:1.13.8 AS dependencies
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
 
-WORKDIR /kubernetes-image-puller
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8/go-toolset
+FROM registry.access.redhat.com/ubi8/go-toolset:1.13.4 as builder
```

Comment:
1.13.4-32

Reply:
@nickboldt does it mean that rhel images in the upstream are not properly supported? (my understanding that 1.13.4 has volnerability)

Reply:
it means CRW-598 isn't done yet. :)

Reply:
note that in https://github.com/che-incubator/kubernetes-image-puller-operator/pull/34/files#diff-4bb306eddc70a07a963059d2c51ad097R16 you're using the public rhel7-based golang 1.13 image (`registry.access.redhat.com/devtools/go-toolset-rhel7:1.13.4-181), but here you're using the rhel-8 go-toolset image. 

Interesting... seems that the rhel8 one has been cross-posted to the ubi8 org, so it too is available w/o authentication.